interfaces/builtin/broadcom_asic_control.go

```diff
@@ -49,9 +40,11 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 `
 
 const broadcomAsicControlConnectedPlugUDev = `
-KERNEL=="linux-user-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-kernel-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-bcm-knet", TAG+="###SLOT_SECURITY_TAGS###"
+# This file contains udev rules for accessing broadcom asic kernel module.
+#
+# Do not edit this file, it will be overwritten on updates
+
+SUBSYSTEM=="net", KERNEL=="bdev", TAG+="%[1]s"
```

Comment:
+1 on the udev changes. The apparmor policy is too broad. Can you give the specific denials the prompted each rule?